Remotion/Web/Development.WebTesting/HostingStrategies/AspNetDockerContainerWrapper.cs

```diff
@@ -0,0 +1,43 @@
+// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Remotion.Web.Development.WebTesting.HostingStrategies.DockerHosting;
+
+namespace Remotion.Web.Development.WebTesting.HostingStrategies;
+
+/// <summary>
+/// Represents an Asp.Net Core Docker container and manages its lifecycle.
+/// </summary>
+public class AspNetDockerContainerWrapper : DockerContainerWrapperBase
+{
+  private readonly string _processPath;
+
+  public AspNetDockerContainerWrapper (IDockerClient docker, AspNetDockerContainerConfigurationParameters configurationParameters)
+      : base(docker, configurationParameters)
+  {
+    _processPath = configurationParameters.ProcessPath ?? throw new ArgumentException(
+        "Executable path was not set while trying to host with ASP.NET core. Please add 'processPath' property to the testSiteLayout configuration parameter.");
```

Comment:
Given the length of the line, I would prefer to split this into an argument check and an assignment. Am not sure that combining these actually makes the code easier to read.